_stbt/transition.py

```diff
@@ -58,7 +57,7 @@ def press_and_wait(
 
     t = _Transition(region, mask, timeout_secs, stable_secs, _dut)
     result = t.press_and_wait(key)
-    sys.stderr.write("%s\n" % result)
+    stbt.debug(str(result))
```

Comment:
This logging could be better.  Perhaps:

    stbt.debug("press_and_wait(%r) -> %r" % (key, result))